app/modules/annotations/models.py

```diff
@@ -181,6 +192,13 @@ def get_individual(self):
             individual = self.encounter.individual
         return individual
 
+    def get_sighting(self):
```

Comment:
i have a `get_sighting_guid()` in my PR, but i think these will live along side of each other just fine.